### PR TITLE
Script for live sentiment analysis

### PR DESCRIPTION
**Pull request by Demborg**
Adds a script for live sentiment analysis and visualization

### CODE REVIEW

File: live_sentiment.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import numpy as np
+
+import models
+import utils
+import settings
+from colored import fg, bg, stylize 
+
+def get_live_sentiment(model, feature):
+    """Takes a trained model and a list of features and returns the
+    estimated scores for each timestep"""
```
Comment (netrome) [inline live_sentiment.py:14]: Nice docstring :)

File: live_sentiment.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    """Takes a rating from 0 to 5 and converts that to a grey scale x-term
+    safe color"""
+
+    val = int(rating/5. * 23 + 232)
+    val = max(0, min(255, val))
+    return val
+
+if __name__ == "__main__":
+    # load model
+    model = models.SimpleLSTM(settings.HIDDEN_SIZE)
```
Comment (netrome) [inline live_sentiment.py:32]: Now we are starting to hardcode the model in lots of places. Perhaps it is time to add a "CURRENT_MODEL" in settings.

**Review by netrome — COMMENTED** [review]
Looking good!

File: live_sentiment.py (hunk 2 above)
Comment (Demborg) [reply live_sentiment.py:32]: yeah, thats true, should be done as a separate PR though right?

**Pull request merged**